pkg/util/kubernetes/apiserver/types.go

```diff
@@ -26,11 +26,11 @@ const (
 type InformerName string
 
 const (
-	endpointsInformer InformerName = "endpoints"
+	endpointsInformer InformerName = "v1/endpoints"
 	// SecretsInformer holds the name of the informer
-	SecretsInformer InformerName = "secrets"
+	SecretsInformer InformerName = "v1/secrets"
 	// WebhooksInformer holds the name of the informer
-	WebhooksInformer InformerName = "webhooks"
+	WebhooksInformer InformerName = "admissionregistration.k8s.io/v1/mutatingwebhookconfigurations"
 	// ServicesInformer holds the name of the informer
-	ServicesInformer InformerName = "services"
+	ServicesInformer InformerName = "v1/services"
```

Comment:
while this change makes sense, its related to container-integration and not our code usage, right?

Reply:
this is to harmonize informers names with the new one we use, in case we want to work on an informer wrapper later

Reply:
makes sense to me